src/lib/entities/audio/model/audio-instance.ts

```diff
@@ -42,3 +43,29 @@ function clampTime(time: number) {
   const d = get(duration);
   return Math.max(0, Math.min(time, isNaN(d) ? 0 : d));
 }
+
+export function reportError(el: HTMLElement) {
+  // shameful workaround
+  if (!window.navigator.onLine) {
+    snackbar({ text: 'You are offline. Please connect to the internet and try again.' });
+    return;
+  }
```

Comment:
I think this should be moved after the `if (err == null) return` line, because why would we display it if there are no errors?

Reply:
Also, it isn't so shameful)